lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -1111,7 +1127,7 @@ const VtDictionary& UsdMayaJobExportArgs::GetDefaultDictionary()
         d[UsdMayaJobExportArgsTokens->exportAssignedMaterials] = true;
         d[UsdMayaJobExportArgsTokens->legacyMaterialScope] = false;
         d[UsdMayaJobExportArgsTokens->exportDisplayColor] = false;
-        d[UsdMayaJobExportArgsTokens->exportDistanceUnit] = true;
+        d[UsdMayaJobExportArgsTokens->exportDistanceUnit] = false;
```

Comment:
Since the default for the new unit option is "Maya Prefs", we are better off with exportDistanceUnit being off by deault, otherwise it would overwrite what the unit option would try to do.